src/aleph_client/chains/tezos.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict
+from aleph_pytezos.crypto.key import Key
+
+from nacl.signing import SigningKey
+from nacl.public import PublicKey, PrivateKey, SealedBox
+
+from .common import (
+    BaseAccount,
+    get_fallback_private_key,
+    get_verification_buffer,
+    get_public_key,
+)
+
+class TezosAccount(BaseAccount):
+    CHAIN = "TEZOS"
+    CURVE = "secp256k1"
+    _account: Key
+
+    def __init__(self, private_key=None):
+        self.private_key = private_key
+        self._account = Key.from_secret_exponent(self.private_key)
+        self._signing_key = SigningKey(self.private_key)
+        self._private_key = self._signing_key.to_curve25519_private_key()
+
+    async def sign_message(self, message: Dict) -> Dict:
+        """Sign a message inplace.
+        """
+        message = self._setup_sender(message)
+
+        verif = get_verification_buffer(message)
+        sig = self._account.sign(verif)
+        message["signature"] = sig
+        return message
+
+    def get_address(self) -> str:
+        return self._account.public_key_hash()
+
+    def get_public_key(self) -> str:
+        return self._account.public_key()
+
+    async def encrypt(self, content) -> bytes:
+        value: bytes = bytes(SealedBox(self._private_key.public_key).encrypt(content))
```

Comment:
The type annotation seems superfluous, and I wonder if you need to cast to bytes as well. You don't do it for the decrypt function?